browser/ui/sidebar/sidebar_browsertest.cc

```diff
@@ -1343,4 +1344,78 @@ IN_PROC_BROWSER_TEST_F(SidebarBrowserTest, SidebarRightSideTest) {
   EXPECT_TRUE(prefs->GetBoolean(prefs::kSidePanelHorizontalAlignment));
 }
 
+// Test browser launching and proper object instantiation for
+// mobile view feature enable state.
+class SidebarBrowserTestWithMobileViewFeature
+    : public testing::WithParamInterface<bool>,
+      public SidebarBrowserTest {
+ public:
+  SidebarBrowserTestWithMobileViewFeature() {
+    if (FeatureEnabled()) {
```

Comment:
this needs to be tested with and without the feature enabled, it should not be dependent on the current state of the feature